flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -70,6 +73,7 @@ public static Builder forRowData() {
     private Table table;
     private TableLoader tableLoader;
     private TableSchema projectedSchema;
+    private ReadableConfig flinkConf;
```

Comment:
For those users that write flink batch jobs in Java API ,  they will always pass a flink's `Configuration`, right ?   So how about defining this as  `org.apache.flink.configuration.Configuraiton`  ?

Reply:
When we construct the `IcebergTableSource`, we use the `TableSourceFactory.Context#getConfiguration` method to get the configuration. This method returns a ReadableConfig, so we use ReadableConfig instead of Configuration. In addition, Configuration is the implementation class of the ReadableConfig interface, so I think ReadableConfig should not has a problem

Reply:
Got your point,  I'd prefer to use flink's `Configuration`  because  it will be exposed to flink developers as an API in  `FlinkSource`,  using the unified `Configuration` will be more straightforward for them.  But as you said,  the  TableSourceFactory#Context is exposing the `ReadableConfig`,  I also did not find a correct way to convert `ReadableConfig` to `Configuration`.   OK, I think we could use `ReadableConfig` here,  thanks.